src/exam/ExamWrapper.jsx

```diff
@@ -8,13 +9,18 @@ import ExamStateContext from '../context';
  */
 const ExamWrapper = ({ children, ...props }) => {
   const state = useContext(ExamStateContext);
+  const { authenticatedUser } = useContext(AppContext);
   const { sequence, courseId } = props;
   const { getExamAttemptsData, getAllowProctoringOptOut } = state;
   const loadInitialData = async () => {
     await getExamAttemptsData(courseId, sequence.id);
     await getAllowProctoringOptOut(sequence.allowProctoringOptOut);
   };
 
+  if (!authenticatedUser) {
```

Comment:
Can we add comment explaining why we do this?

```suggestion
// if the user is browsing public content (not logged in) they cannot be in an exam
// any requests for exam state will 403 so just short circuit this component here
  if (!authenticatedUser) {
```